Description: This test case makes assumptions about the order
 of execution of tests in the Fruit.class test which are not
 guaranteed under Java 7.
 .
 This is being worked on upstream for a future release on JUnit.
Author: James Page <james.page@ubuntu.com>
Forwarded: not-needed

Index: junit4/src/test/java/org/junit/tests/running/classes/ParentRunnerTest.java
===================================================================
--- junit4.orig/src/test/java/org/junit/tests/running/classes/ParentRunnerTest.java	2012-05-21 14:04:38.165350000 +0100
+++ junit4/src/test/java/org/junit/tests/running/classes/ParentRunnerTest.java	2012-05-21 14:29:26.162439110 +0100
@@ -9,6 +9,7 @@
 
 import org.hamcrest.Matcher;
 import org.junit.Test;
+import org.junit.Ignore;
 import org.junit.internal.matchers.TypeSafeMatcher;
 import org.junit.runner.Description;
 import org.junit.runner.JUnitCore;
@@ -38,6 +39,7 @@
 		}
 	}
 
+    @Ignore
 	@Test
 	public void useChildHarvester() throws InitializationError {
 		log= "";
